LiteCore/Storage/UnicodeCollator.cc

```diff
@@ -50,6 +50,48 @@ namespace litecore {
         });
     }
 
+    bool LikeUTF8(slice comparand, slice pattern, const Collation& col) {
```

Comment:
It doesn't look like this handles backtracking in the pattern, i.e. false matches. For example, can it handle `'(oo)oo)' LIKE '(%)' ?

Reply:
You mean % is supposed to be greedy?